scripts/e2e/test.sh

```diff
@@ -82,9 +82,14 @@ if [ "x${DATASET_URL}" == "x" ]; then
   export DATASET_URL="http://dataset-dummy-server-e2e:8000";
   export DATASET_API_KEY="dummy";
 fi;
-log "DATASET_URL: ${DATASET_URL}";
-log "DATASET_API_KEY: XXXXX${DATASET_API_KEY: -4}";
-log "TEST_RUN_SERVERHOST: ${TEST_RUN_SERVERHOST}"
+log "DATASET_URL: '${DATASET_URL}'";
+log "DATASET_API_KEY: '${DATASET_API_KEY: -4}' (last 4 characters)";
+log "TEST_RUN_SERVERHOST: '${TEST_RUN_SERVERHOST}'"
+
+if [ "x${DATASET_API_KEY}" == "x" ]; then
```

Comment:
why not simply? Maybe I am missing something
```suggestion
if [ "${DATASET_API_KEY}" == "" ]; then
```

Reply:
If you want to simplify it, there is also the `if [ -z "${DATASET_API_KEY}" ]; then ....`

I do not know. These is even stack overflow question about it - https://serverfault.com/questions/7503/how-to-determine-if-a-bash-variable-is-empty. I have used same construct few lines above this.

It's hard to google why, but ChatGPT matches my believes:
![Screenshot 2023-11-21 at 10 27 40](https://github.com/scalyr/opentelemetry-exporter-dataset/assets/122797378/80cc9e30-1ca6-436c-9aeb-583eeca43f1c)